### PR TITLE
Update Onboarding Flow with Elements

### PR DESCRIPTION
**Pull request by ghost**
Hi, this is Siddharth from the [LeapWallet](https://twitter.com/leap_cosmos) team!

The current onboarding flow shows the user that they need 5 AKT or USDC, but users have to obtain them by jumping through hoops. 

Elements gives the users 2 simple ways to obtain AKT tokens on Akash-
1. Swap any other cosmos token for AKT
2. Swap EVM Tokens for AKT

You can check out the demo here - https://cloudmos-elements.vercel.app/

Demo Video - 

https://github.com/akash-network/cloudmos/assets/114133109/0a8b3886-5eb8-441e-b60b-782e35fa9eda

### CODE REVIEW

**Comment by ghost** [comment]
@baktun14 Please take a look at this, thanks!

File: deploy-web/package.json [hunk 1]
```diff
@@ -93,6 +95,6 @@
     "@types/react-simple-maps": "^3.0.0",
     "patch-package": "^8.0.0",
     "prettier": "^3.0.2",
-    "typescript": "^4.5.5"
+    "typescript": "^5.3.2"
```
Comment (baktun14) [inline deploy-web/package.json:98]: Any reason you updated typescript?

File: deploy-web/src/context/WalletProvider/WalletProvider.tsx [hunk 2]
```diff
@@ -91,7 +91,9 @@ export const WalletProvider = ({ children }) => {
   }, []);
 
   useEffect(() => {
-    if (isWindowLoaded && isSettingsInit) {
+    // removing isSettingsInit as the wallet status keeps loading
```
Comment (baktun14) [inline deploy-web/src/context/WalletProvider/WalletProvider.tsx]: I think we have to put it back

**Comment by baktun14** [comment]
This looks great!

File: deploy-web/src/components/liquidity-modal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    primaryButtonText: "#ffffff",
+    alpha: "#D6D6D6",
+    gray: "#9E9E9E",
+    error: "#ff6961",
+    errorBackground: "#272727",
+    success: "#29A874",
+    successBackground: "#0D3525",
+    warning: "#ffa726"
+  },
+  zIndices: defaultZIndices,
```
Comment (Redm4x) [inline deploy-web/src/components/liquidity-modal/index.tsx]: The website header has a zIndex of `1100` and goes over the modal if it uses the `defaultZIndices` (`1000`).
```suggestion
  zIndices: { modalOverlay: "1200" },
```

File: deploy-web/package.json (hunk 1 above)
Comment (ghost) [reply deploy-web/package.json:98]: Yes, there was a build issue - https://github.com/wevm/abitype/issues/160

File: deploy-web/src/context/WalletProvider/WalletProvider.tsx (hunk 2 above)
Comment (ghost) [reply deploy-web/src/context/WalletProvider/WalletProvider.tsx]: Yes, I will put it back

File: deploy-web/src/components/liquidity-modal/index.tsx (hunk 3 above)
Comment (ghost) [reply deploy-web/src/components/liquidity-modal/index.tsx]: Thanks for the note!

**Comment by ghost** [comment]
Thanks @Redm4x @baktun14 for your reviews, I will resolve the comments and push the changes asap.

File: deploy-web/package.json (hunk 1 above)
Comment (baktun14) [reply deploy-web/package.json:98]: Hmm ok, there doesn't to have any impacting breaking changes.

**Pull request merged**